app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/SessionsFragment.kt

```diff
@@ -122,7 +122,9 @@ class SessionsFragment : DaggerFragment(), Findable, OnReselectedListener {
                 is Result.Success -> {
                     sessionsViewPagerAdapter.setSessionTab(result.data)
                     activity?.invalidateOptionsMenu()
-                    if (Prefs.enableReopenPreviousRoomSessions and (savedInstanceState == null)) {
+                    if ((result.data is SessionTab.Room) and
```

Comment:
Before this app opened `PreviousOpenedItem` when move from another page like favarite tab.
I stoped it at the schedule tab.